examples/disctl/build.bash

```diff
@@ -0,0 +1,101 @@
+##### DEFAULTS #####
+
+# Set some global default configurations. These just tell discovery what
+# container filesystems to use for the endpoint nodes and the router.
+bin/disctl -update-config minicccfs /opt/minimega/images/miniccc
+bin/disctl -update-config minirouterfs /opt/minimega/images/minirouter
+
+##### BUILD GRAPH #####
+
+# Create first network (we'll add two endpoints to it later). The value
+# returned by the bin/disctl command is the ID of the network, which we'll
+# need later to add endpoints to it.
+net0=$(bin/disctl -nn)
+
+# Create the two endpoints that will be connected to the first network.
+# The value returned by the bin/disctl command is the ID of the endpoint,
+# which we'll need later to add config options and add it to the
+# network.
+
+# Create the first endpoint.
+endpoint=$(bin/disctl -ne)
+
+# Add the first endpoint to the network.
+bin/disctl -c $net0 $endpoint
+
+# Create the second endpoint.
+endpoint=$(bin/disctl -ne)
+
+# Add the second endpoint to the network.
+bin/disctl -c $net0 $endpoint
+
+# Create second network (we'll add two endpoints to it later). The value
+# returned by the bin/disctl command is the ID of the network, which we'll
+# need later to add endpoints to it.
+net1=$(bin/disctl -nn)
+
+# Create the two endpoints that will be connected to the second network.
+# The value returned by the bin/disctl command is the ID of the endpoint,
+# which we'll need later to add config options and add it to the
+# network.
+
+# Create the first endpoint.
+endpoint=$(bin/disctl -ne)
+
+# Add the first endpoint to the network.
+bin/disctl -c $net1 $endpoint
+
+# Create the second endpoint.
+endpoint=$(bin/disctl -ne)
+
+# Add the second endpoint to the network.
```

Comment:
Do you want to change this to be a for loop? Something like:

```bash
for i in $(seq 2); do
	# Create an endpoint.
	endpoint=$(bin/disctl -ne)
	# Add it to the network.
	bin/disctl -c $net1 $endpoint
done
```

This makes it clearer how you might extend this to many nodes.

Reply:
Great idea. Can do!